src/Snowflake/ConnectionExceptionResolver.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Db\Import\Snowflake;
+
+use Keboola\Db\Import;
+
+class ConnectionExceptionResolver
+{
+    public static function resolveException(\Throwable $e): void
+    {
+        if (get_class($e) === \ErrorException::class) {
```

Comment:
ten `\ErrorException` mě překvapil ale je to daný vlastně tím že ta connection class spoléhá na nahozený error handler takže v téhle verzi to jinak nepůjde. V nové separátní connection class se to musí vyřešit tak at to vyhazuje vlastní výjimky https://github.com/keboola/php-db-import/issues/48 a součástí toho bude tedy i tenhle exception resolver.